### ISSUE TITLE
crashing to the point of unusability?

### ISSUE THREAD

**Issue by ego-rock** [stakira/OpenUtau#658]
### Acknowledgement

- [X] I have read Getting-Started and FAQ

### 🐛 Describe the bug

hi! i've been using openutau for a good bit now and have never had this issue before. just today i updated to v0.1.56 and i saw that when adding in an instrumental .wav, it just... wouldn't play. i tried this with different audio to see if it would work and it didn't.
then i saw that every time i added a singer and added any note+syllable, as soon as i picked a syllable the application would crash. i tried this with multiple different singers and it happened every single time, only after adding a syllable to a note. the application is able to be opened, it just... won't play audio and crashes upon adding syllables to a singer's note. this has never happened until installing the latest version. i'm on a macbook pro m1, big sur. thank you for your hard work on this :)

### Explains how to reproduce the bug

1. install latest version of openutau, v0.1.56.0
2. add any singer to a project 
3. add a part for the singer
4. add any note
5. assign any syllable to the note
6. application crashes after a couple seconds

### OS & Version

macOS Big Sur 11.0

### Logs

[OpenUtau_2023-04-14-162924_MacBook-Pro-461.txt](https://github.com/stakira/OpenUtau/files/11236768/OpenUtau_2023-04-14-162924_MacBook-Pro-461.txt)

**Comment by alex9505** [comment]
That log does not look like the usual OpenUtau log file. Is it the log file located in the installation folder's "Logs" folder?
Have you tried changing the "Audio Backend" option under the "Playback Device" option?
On what version did it work?

This is what the log file looks like for me, not sure why it would be so different on MacOS:
[log20230420.txt](https://github.com/stakira/OpenUtau/files/11289349/log20230420.txt)

**Comment by alex9505** [comment]
It looks like there was another issue that was opened on this with the same problem. Link: #650

**Comment by BagelHero** [comment]
This was a bug, and should now be fixed in the latest version. Please let us know if it remains an issue, and if you need to roll back the last working release should be 0.1.52!

**Comment by morgansleeper** [comment]
Just wanted to say I was having this issue (with picking a syllable crashing the app every time), and the latest update fixes it for me! Thanks so much!!
